src/small-component.scss

```diff
@@ -4,6 +4,142 @@
  */
 
+
+.gc-form {
+	clear: both;
+	padding-top: 6px;
+}
+.gc-form {
+	.checkbox, .radio, .form-checkbox, .form-radio {
+		-moz-user-select: none;
+		-ms-user-select: none;
+		-webkit-user-select: none;
+		cursor: pointer;
+		font-weight: normal;
+		position: relative;
+		user-select: none;
+	}
+	.checkbox, .radio {
+		margin-bottom: 35px;
+		padding-left: 30px;
+	}
+	.checkbox input[type="checkbox"]:focus + label::before, .form-checkbox input[type="checkbox"]:focus + label::before, .form-radio input[type="radio"]:focus + label::before, .radio input[type="radio"]:focus + label::before	{
+		box-shadow: 0 0 3pt 2pt rgb(59, 153, 252);
+	}
+	.checkbox label::after, .form-checkbox label::after {
+		-ms-transform: rotate(45deg);
+		-webkit-transform: rotate(45deg);
+		border: solid #333;
+		border-width: 0 5px 5px 0;
+		content: "";
+		display: inline-block;
+		height: 22px;
+		left: 12px;
+		position: absolute;
+		top: 0px;
+		transform: rotate(45deg);
+		width: 13px;
+	}
+	.checkbox label::before, .radio label::before, .form-checkbox label::before, .form-radio label::before {
+		border: 2px solid #333;
+		box-sizing: border-box;
+		content: "";
+		display: inline-block;
+		height: 36px;
+		left: 0;
+		margin-top: -4px;
+		position: absolute;
+		top: 0;
+		width: 36px;
+	}
+	.checkbox-green input[type="checkbox"]:checked + label:before {
+		background-color: #318000;
+	}
+	.checkbox-green label::after {
+		border: solid white;
+		border-width: 0 5px 5px 0;
+		content: "";
+	}
+	.checkbox-small {
+		margin-bottom: 25px;
+		padding-left: 16px;
+	}
+	.checkbox-small label {
+		font-size: 18px !important;
+	}
+	.checkbox-small label::after {
+		-ms-transform: rotate(45deg);
+		-webkit-transform: rotate(45deg);
+		border: solid #333;
+		border-width: 0 3px 3px 0;
+		content: "";
+		display: inline-block;
+		height: 14px;
+		left: 8px;
+		position: absolute;
+		top: 2px;
+		transform: rotate(45deg);
+		width: 8px;
+	}
+	.checkbox-small label::before {
+		border: 2px solid #333;
+		box-sizing: border-box;
+		content: "";
+		display: inline-block;
+		height: 24px;
+		left: 0;
+		margin-top: 0;
+		position: absolute;
+		top: 0;
+		width: 24px;
+	}
+	.form-checkbox, .form-radio {
+		margin-bottom: 25px;
+		margin-right: 30px;
+		min-width: 26%; /* prevents weird overlapping */
+	}
+	.form-checkbox input[type="checkbox"] + label::after, .form-radio input[type="radio"] + label::after, .checkbox input[type="checkbox"] + label::after, .radio input[type="radio"] + label::after {
+		content: none;
+	}
+	.form-checkbox label, .form-radio label {
+		padding-left: 50px;
+	}
+	input {
+		opacity: 0;
+		position: absolute;
+	}
+	label {
+		font-size: 20px;
+	}
+	.radio input[type="radio"]:checked + label::after, .checkbox input[type="checkbox"]:checked + label::after, .form-radio input[type="radio"]:checked + label::after, .form-checkbox input[type="checkbox"]:checked + label::after, {
+		content: "";
+	}
+	.radio label::after, .form-radio label::after {
```

Comment:
More a note for myself for testing. I wonder if this will interfere with implicit label for input element.